source/install_scripted.rst

```diff
@@ -127,6 +127,8 @@ Options:
     - Contains the fully qualified path to the installation directory for the Quipucords server. Defaults to ``~/quipucords/``.
  - **server_port**
     - Contains the port number for the Quipucords server. Defaults to ``9443``.
+ - **open_port**
+    - Contains a ``true`` or ``false`` value. Defaults to ``true``. Supply ``false`` to skip opening the server port on the machine running the server. Opening the port will be skipped if the installation script is not running with escalated privileges.
```

Comment:
I would say "Supply `false` to install without opening the server port in the firewall." instead of "Supply `false` to skip opening the server..." Also, I'm not sure we need to include the part about escalated privileges, because if you look at the `qpc_server` role in the install, many of the commands require become privileges. But if we do want to include that, I would reword it to say "The installation script must be ran with elevated privileges to open the server port."

Reply:
Minor remark on @abaiken comment: 
...installation script must be run....
BUT better to skip fancy verb tenses to avoid bouncing around the space-time continuum, so use this instead....
...installation script must run...